src/js/components/navigation.js

```diff
@@ -7,7 +7,6 @@ const accordion = require('./accordion');
 const CLICK = require('../events').CLICK;
 const PREFIX = require('../config').prefix;
 
-const CONTEXT = 'header';
```

Comment:
I got rid of this and the focus logic that required the close button be inside it because the close button *isn't* inside a `<header>` on standards.usa.gov, which meant that the close button wasn't even being focused on that site after the user activated the menu. 😞